setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import setuptools
+
+with open("README.md", "r") as readme:
+    long_description = readme.read()
+
+deps = os.popen('pip freeze').readlines()
+# readlines() doesn't strip trailing newlines
+deps = [d.strip() for d in deps]
+
+setuptools.setup(
+    name="net-analysis",
```

Comment:
Do we want to give this a clearer or more jigsaw/firehook related name before putting it in pypi forever? Maybe just jigsaw-net-analysis?

Reply:
is the firehook name external?

Reply:
Good point, even though we may not end up put in pypi. Firehook is not really external. 
I like the `jigsaw-net-analysis` suggestion.

Reply:
Done in 8d66cb7